src/modules/github/github.actions.js

```diff
@@ -0,0 +1,21 @@
+import { LOAD_ALL_REPOS, SEARCH_REPOS, CLEAR } from "./github.types";
+
```

Comment:
Tutaj możemy wykorzystać dane z github.api.js, aby móc stworzyć `thunka`.
Jeśli chodzi o materiały to tam na stronie 62 znajdziesz taki kod:
```
// ./src/actions/ip.js
const url = 'https://api.ipify.org?format=json';
// ...
export const getIP = () => {
 return function thunk(dispatch, getState) {
     return fetch(url)
         .then( resp => {
             if(resp.ok) { return resp.json(); }
                 throw new Error('Err!');
          })
         .then( resp => dispatch(setIP( resp )))
         .catch( err => dispatch(addError( err )))
 }
}
```
W naszym przypadku mogłoby to wyglądać tak:
```
import {loadRepos} from 'github.api.js';

export const getFromApiAction = () => {
    return function thunk(dispatch, getState) { 
        // uruchamiam pierwszą akcję, która pobiera dane z API
        loadRepos.then(repos => {
            // tutaj uruchamiam drugą akcję, która ładuje dane
            // zwróć uwagę, że ona jest zdefiniowana poniżej
            dispatch(loadAllReposAction(repos))
        }
    }
}
```